docs/migrating/migrating_from_ez_publish_platform.md

```diff
@@ -65,17 +74,17 @@ Assuming you have own composer packages *(libraries and bundles, but not eZ Publ
 
 `composer require --no-update "vendor/package:~1.3.0"`
 
-Adapt the command with your `vendor`, `package`, version number, and add `"–dev"` if a given package is for dev use. Also check if there are other changes in `composer.json` you should move over.
+Adapt the command with your `vendor`, `package`, version number, and add `"–dev"` if a given package is for dev use only. Also check if there are other changes in `composer.json` you should move over.
 
 ###### 2.2.2 Install XmlText Field Type
 
 While no longer bundled, the XmlText Field Type still exists and is needed to perform a migration from eZ Publish's XmlText to the new docbook-based format used by the RichText Field Type. If you plan to use Legacy Bridge for a while before migrating content, you'll also need this for rendering content with XMLText. From `<new-ez-root>` execute:
 
-`composer require --no-update "ezsystems/ezplatform-xmltext-fieldtype:^1.3.0"`
+`composer require --no-update "ezsystems/ezplatform-xmltext-fieldtype:^1.7.3"`
```

Comment:
Blocker: This version does not support 1.7LTS: https://github.com/ezsystems/ezplatform-xmltext-fieldtype/blob/v1.7.3/composer.json#L13